telemetry/telemetry_record.py

```diff
@@ -62,10 +69,30 @@ def write_page_sets(urls):
         template = ''.join(f.readlines())
         assert template, 'Failed to read from page set template'
     for i in range(len(urls)):
-        test_name = 'test{0}'.format(i)
+        test_name = 'url{0}'.format(i)
+        with open('{0}{1}.py'.format(page_set_path, test_name), 'wb') as f:
+            f.write(template.format(test_name, urls[i]))
+        # Write modified version
+        test_name = 'url{0}'.format('999999' + str(i))
         with open('{0}{1}.py'.format(page_set_path, test_name), 'wb') as f:
             f.write(template.format(test_name, urls[i]))
 
+def store_url_mapping(url_list):
+    """Stores a mapping from index to url to be used later.
+
+    Saved to data/url_mapping.db: {index: url,...}
+    :param url_list: a list of each url to store
+    """
+    url_mapping = {}
+    for i in range(len(url_list)):
+        url_mapping[i] = url_list[i]
+        # Write modified version
+        url_mapping[int('999999' + str(i))] = url_list[i]
```

Comment:
Curious: if you store the url in the filename itself (as a base 64 encoded string), would you need this? You could just do a glob.glob() to find the file again, given the url.